### PR TITLE
Travel Fund for Open Source Africa

### PR DESCRIPTION
**Pull request by codeekage**
I'd be facilitating this event: elaborate here: https://github.com/nodejs/admin/issues/390

Stipends: 
- Air Travel - $132.66
- Hotel - $254.26

Total - $386.92

### CODE REVIEW

**Comment by Trott** [comment]
In general, I'm in favor of this given both the reasonable cost compared to other travel we fund and our need to do a lot better with outreach in pretty much all of Africa. However, a few more details would be helpful:

* In accordance with the [travel fund policy requirements](https://github.com/nodejs/admin/blob/master/MEMBER_TRAVEL_FUND.md#requirements), neither your employer nor the event would be willing to pay for this, correct?

* Can you provide just a little bit more detail: What does "facilitate" mean here? The event is a general open source event, right? Is there anything Node.js-specific that you will be bringing to it? (Not sure that matters as much as it once did since the travel fund will be an OpenJS thing rather than a Node.js thing probably within six months.)

Thanks for wanting to do this!

**Comment by Trott** [comment]
(Also, to be clear, you don't need to be an organizer/facilitator/speaker/panelist to qualify for travel funds. It is permissible according to the policy to use the travel fund for simple conference attendance. So if what "facilitate" means here is closer to "participate", that should still be within bounds.)

**Comment by codeekage** [comment]
> In accordance with the travel fund policy requirements, neither your employer nor the event would be willing to pay for this, correct?

I'm currently not affiliated with any company, and the event is quite a starter and paying for travels won't be possible at this time. 

> Can you provide just a little bit more detail: What does "facilitate" mean here?

I'd be giving talks on Node.js Open Source, and Open Source Culture (in general). Will help folks get started with OSS bounties.

**Comment by Trott** [comment]
> I'd be giving talks on Node.js Open Source, and Open Source Culture (in general). Will help folks get started with OSS bounties.

Cool! Does your Node.js talk have a title or is there a slide deck?

**Comment by ahmadawais** [comment]
+1 for the support on this PR. I know @codeekage would do an awesome job at this, he's been quite active in the African region w.r.t Node.js.

File: TravelFunds/2019.md [hunk 1]
```diff
@@ -20,3 +20,4 @@ Jory | Burson| Collaborator Summit | Attendance / Co-Organizing || Berlin | 28 M
 Anna | Henningsen | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 257.25€ | 29 Apr 2019 | ||286.45|26 Jun 2019|12 Jul 2019|$300
 Glenn | Hinks | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | US $1660 | 25 Apr 2019 |||Withdrawn|N/A|N/A|N/A
 Mykola | Grybyk | Collaborator Summit | Attendance || Berlin | 30 May - 31 May 2019 | 320.57€ | 2 May 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
+Abraham | Agiri | OSCA (Open Soruce Africa) | Facilitator || Lagos NG | 1 - 4 August 2019 | $386.92 | 24 July, 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
```
Comment (nschonni) [inline TravelFunds/2019.md]:
```suggestion
Abraham | Agiri | OSCA (Open Source Africa) | Facilitator || Lagos NG | 1 - 4 August 2019 | $386.92 | 24 July, 2019 | ||27 Jun 2019|€320.57|12 Jul 2019|$365
```

**Comment by Trott** [comment]
@codeekage Can you please indicate just a little more detail about what you'll be doing. You said you'll be giving a talk about Node.js. What will it be about? An introduction to Node.js? An overview of the project and new features coming? Something else?

**Comment by codeekage** [comment]
>@codeekage Can you please indicate just a little more detail about what you'll be doing. You said you'll be giving a talk about Node.js. What will it be about? An introduction to Node.js? An overview of the project and new features coming? Something else?

My talk will be on Open Source Culture and Node.js as it's a case study. How one can become a Node Contributor or Collaborator without code contribution. 

Many folks believe to contribute to Open Source it always has to be by codes, or something more difficult. Will be sharing my experience as a CommComm member and how I've been a collaborator with 0 code contributions.

**Comment by Trott** [comment]
> Many folks believe to contribute to Open Source it always has to be by codes, or something more difficult. Will be sharing my experience as a CommComm member and how I've been a collaborator with 0 code contributions.

Oh, that's great! I would love to see this talk (or a slide deck or a recording) some time!

Hey, @nodejs/community-committee! This needs one more CommComm approval (and another 11 hours).

**Comment by Trott** [comment]
Also pinging @nodejs/tsc (although this already has sufficient TSC approval--but pile-on approvals never hurt).

**Comment by ahmadawais** [comment]
@trott done! 👍

**Comment by Trott** [comment]
@nodejs/community-committee! This still needs one more CommComm approval.

**Comment by Trott** [comment]
> @nodejs/community-committee! This still needs one more CommComm approval.

The @nodejs/community-committee membership does not match up with membership listed at https://github.com/nodejs/community-committee#community-committee-members. This is creating ambiguity here.

**Comment by codeekage** [comment]
@Trott we already got two @nodejs/community-committee approvals @ahmadawais and @WaleedAshraf :)

**Comment by Trott** [comment]
> @Trott we already got two @nodejs/community-committee approvals @ahmadawais and @WaleedAshraf :)

@ahmadawais is listed in the Community Committee README but is not a member of the @nodejs/community-committee GitHub team. ¯\\_(ツ)_/¯ @amiller-gh

**Comment by Trott** [comment]
https://github.com/nodejs/community-committee/pull/463 makes it pretty clear that @ahmadawais should be added to the GitHub team. No need to make @codeekage wait while that gets sorted out, though. Landing!

**Pull request merged**